src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/Manager/MediaManagerTest.php

```diff
@@ -298,6 +299,11 @@ public function testSaveWrongVersionType()
         $this->mediaManager->save($uploadedFile->reveal(), ['id' => 1], 42);
     }
 
+    public function testVideoUploadWithoutFFmpeg()
+    {
+        $this->ffprobe->format(Argument::any())->willThrow(ExecutableNotFoundException::class);
```

Comment:
This way you haven't really tested anything. You just said how the `format` method should behave. You still need a call on the `MediaManager`.